src/main/java/roomescape/client/PaymentClient.java

```diff
@@ -0,0 +1,34 @@
+package roomescape.client;
+
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClient;
+import roomescape.dto.reservation.TossPaymentConfirmRequestDto;
+
+import java.util.Base64;
+
+@Component
+public class PaymentClient {
+
+    private static final String PAYMENT_CONFIRM_URL = "https://api.tosspayments.com/v1/payments/confirm";
+    private static final String PAYMENT_CONFIRM_SECRET_KEY = "test_gsk_docs_OaPz8L5KdmQXkzRz3y47BMw6:";
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BASIC = "Basic ";
+
+    private final RestClient restClient;
+
+    public PaymentClient(final RestClient restClient) {
+        this.restClient = restClient;
+    }
+
+    public void confirmPayment(TossPaymentConfirmRequestDto requestDto) {
+        restClient.post()
+                .uri(PAYMENT_CONFIRM_URL)
+                .header(AUTHORIZATION, BASIC +
```

Comment:
이미 정의된 상수로 대체해볼 수 있을 것 같아요. 😃
 
```suggestion
                .header(HttpHeaders.AUTHORIZATION, BASIC +
```

Reply:
감사합니다!!